grouper/ctl/sync_db.py

```diff
@@ -17,10 +18,36 @@ def sync_db_command(args):
 
     # Add some basic database structures we know we will need if they don't exist.
     session = make_session()
+    # This group is needed to bootstrap a Grouper installation.
+    admin_group = Group.get(session, name="grouper-administrators")
+    if not admin_group:
+        admin_group = Group(
+                groupname="grouper-administrators",
+                description="Administrators of the Grouper system.",
+                canjoin="nobody",
+        )
+
+        try:
+            admin_group.add(session)
+            session.flush()
+        except IntegrityError:
+            session.rollback()
+            raise Exception('Failed to create group: grouper-administrators')
+    admin_group_permission_names = [perm[1] for perm in admin_group.my_permissions()]
```

Comment:
I modified the code in 0fd2a16 to only take action on `grouper-administrators` if it doesn't already exist. This gives the site administrator the ability to disable, &c the group w/o `sync_db` overriding them.